### PR TITLE
Swap forwarder interfaces.

### PR DESCRIPTION
**Pull request by VitalyGushin**
cc @denis-tingaikin @NikitaSkrynnik 

Screenshot 2024-02-06 at 18 25 38

### CODE REVIEW

**Comment by denis-tingaikin** [comment]
@VitalyGushin Could you also attach a picture of how it looks now?

File: model_utils.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2023 Cisco and/or its affiliates.
+// Copyright (c) 2024 Cisco and/or its affiliates.
```
Comment (denis-tingaikin) [inline model_utils.go:1]:
```suggestion
// Copyright (c) 2023-2024 Cisco and/or its affiliates.
```

File: model_utils.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 //
-// Copyright (c) 2023 Pragmagic Inc. and/or its affiliates.
+// Copyright (c) 2024 Pragmagic Inc. and/or its affiliates.
```
Comment (denis-tingaikin) [inline model_utils.go:3]:
```suggestion
// Copyright (c) 2023-2024 Pragmagic Inc. and/or its affiliates.
```

**Review by denis-tingaikin — COMMENTED** [review]
Remote case looks good with the patch. Have you tested it with local cases?

**Comment by VitalyGushin** [comment]
> Remote case looks good with the patch. Have you tested it with local cases?

For local cases interfaces on forwarders also will be swapped

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**